README.md

```diff
@@ -14,5 +14,25 @@ Use the values you obtained in step 2.1 to configure the following variables ins
 1. **projectID**: The project ID (pro-xxxx).
 2. **apiSecret**: The API secret (something cryptic).
 
+2.3 Configuring IOS Associated Domains
+If you are running the IOS App, you need an associated domain file stored on your website, so that the IOS Device can download and verify the domains in your entitelement. For Corbado Service you can use the following JSON template: 
+```json
+{
+  "applinks": {
+    "details": [
+    ]
+  },
+  "webcredentials": {
+    "apps": [ "{ YOUR_APP_BUNDLE_IDENTIFIER }" ]
+  },
+  "appclips": {
+    "apps": []
+  }
+}
+```
+The JSON file needs to be stored under ```https://<your fully qualified domain>/.well-known/apple-app-site-association```.
```

Comment:
kannst du hier noch ein Beispiel angeben? und noch in Klammern dazuschreiben, ob man Port braucht